src/ngraph/op/deformable_psroi_pooling.cpp

```diff
@@ -0,0 +1,157 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "deformable_psroi_pooling.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v1::DeformablePSROIPooling::type_info;
+
+op::v1::DeformablePSROIPooling::DeformablePSROIPooling(const Output<Node>& input,
+                                                       const Output<Node>& coords,
+                                                       const Output<Node>& offsets,
+                                                       const int64_t output_dim,
+                                                       const int64_t group_size,
+                                                       float spatial_scale,
+                                                       const std::string mode,
+                                                       int64_t spatial_bins_x,
+                                                       int64_t spatial_bins_y,
+                                                       bool no_trans,
+                                                       float trans_std,
+                                                       int64_t part_size)
+    : Op({input, coords, offsets})
+    , m_output_dim(output_dim)
+    , m_group_size(group_size)
+    , m_spatial_scale(spatial_scale)
+    , m_mode(mode)
+    , m_spatial_bins_x(spatial_bins_x)
+    , m_spatial_bins_y(spatial_bins_y)
+    , m_no_trans(no_trans)
+    , m_trans_std(trans_std)
+    , m_part_size(part_size)
+{
+    constructor_validate_and_infer_types();
+}
+
+op::v1::DeformablePSROIPooling::DeformablePSROIPooling(const Output<Node>& input,
+                                                       const Output<Node>& coords,
```

Comment:
The specification has been recently updated to have all 3 inputs required. Sorry for the late response, but the MR with update has been committed just a minute ago.

Reply:
Op updated to match new specification